### PR TITLE
[BACKPORT] CBL-6146: Backporting fixes

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Application of feedback RE ticket: https://issues.couchbase.com/browse/CBL-6146

Also added back fullsync snippets kotlin tags that seemed to have perished in a merge.

### CODE REVIEW

**Review by bmeike — APPROVED** [review]
yesssss!

File: modules/ROOT/pages/_partials/commons/common-document.adoc [hunk 1]
```diff
@@ -263,7 +263,7 @@ ifdef::is-android[]
 [NOTE]
 --
 Care should be taken when storing and recovering data in a document or converting that document to JSON and back. +
-Data encoding (Fleece) can result in `Long` values being converted to `Float` instead of `Double`. +
+Data encoding (Fleece) can result in `Long` values being converted to `Integers`, and `Double` values to `Float`. +
```
Comment (simon-dew) [inline modules/ROOT/pages/_partials/commons/common-document.adoc:266]: Should this be just `Integer`?

```suggestion
Data encoding (Fleece) can result in `Long` values being converted to `Integer`, and `Double` values to `Float`. +
```

**Review by simon-dew — APPROVED** [review]
Just one comment, otherwise LGTM. Approved to unblock.

File: modules/ROOT/pages/_partials/commons/common-document.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/_partials/commons/common-document.adoc:266]: You're correct. Will leave tab open and this comment unresolved to fix at a later date

**Pull request merged**